Lib/importlib/_bootstrap_external.py

```diff
@@ -802,7 +802,7 @@ def __init__(self, fullname, path):
         """Cache the module name and the path to the file found by the
         finder."""
         self.name = fullname
-        self.path = path
+        self.path = path.__fspath__() if hasattr(path, '__fspath__') else path
```

Comment:
@brettcannon why is some class meta using `os` at the top, but here we need to use `_os`?